### PR TITLE
Connect 'My Account' to only the user's Review Comments

### PR DESCRIPTION
**Pull request by shreyabarua**
Now when a user is logged in and clicks on 'My Account' they will only see their own comments on their resume.
Moved Log In, Log Out, My Account links out of the navbar and onto the top of the home page. 
Also added a personalized welcome message on home page for any logged in user.

### CODE REVIEW

File: resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java [hunk 1]
```diff
@@ -57,6 +57,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(reviewerEntity);
 
-    response.sendRedirect("resume-review.html");
+    response.sendRedirect("index.html");
```
Comment (astrelni) [inline resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java]: Should this also have the `/`?

File: resume-buddy/src/main/webapp/scripts/script.js [hunk 2]
```diff
@@ -13,19 +13,21 @@ function login() {
...
             // Show the 'my account' option
             // Show a log out option
-            myAccountElement.style.display = "block";
-            loginLinkElement.innerHTML = "<a style=\"text-decoration:none\" href=\"" + 
-            login.logout_url + "\">Log Out</a>";
-            
+            greetingElement.innerHTML = "Welcome " + login.email + "!";
+            myAccountElement.innerHTML = "<a href=\"resume-review.html\">My Account</a>" +
+            "  •  " + "<a href=\"" + login.logout_url + "\">Log Out</a>";
+            loginLinkElement.style.display = "none";
```
Comment (astrelni) [inline resume-buddy/src/main/webapp/scripts/script.js:23]: `#my-account` is currently commented out in the html.

File: resume-buddy/src/main/webapp/index.html [hunk 3]
```diff
@@ -13,16 +13,19 @@
...
       </div>
       <div class="w3-padding-64 w3-large w3-text-grey" style="font-weight:bold">
         <a href="#" class="w3-bar-item w3-button">About Us</a>
         <a href="reviewer-form.html" class="w3-bar-item w3-button">Review</a>
         <a href="reviewee-form.html" class="w3-bar-item w3-button">Get a Review</a>
         <a href="#" class="w3-bar-item w3-button">Meet the Team</a>
-        <a href="#" class="w3-bar-item w3-button" id = "login-container" onclick="login()">Log In</a>
-        <a href="resume-review.html" class="w3-bar-item w3-button" id="my-account">My Account</a>
+        <!--<a href="#" class="w3-bar-item w3-button" id = "login-container" onclick="login()">Log In</a>-->
+        <!--<a href="resume-review.html" class="w3-bar-item w3-button" id="my-account">My Account</a>-->
```
Comment (astrelni) [inline resume-buddy/src/main/webapp/index.html]: Do you not need these anymore and they should just be deleted?

File: resume-buddy/src/main/webapp/scripts/script.js [hunk 4]
```diff
@@ -13,19 +13,21 @@ function login() {
...
         }
         else {
             // Show the log in option
-            loginLinkElement.innerHTML = "<a style=\"text-decoration:none\" href=\"" + 
+            loginLinkElement.innerHTML = "<a href=\"" +
```
Comment (astrelni) [inline resume-buddy/src/main/webapp/scripts/script.js:27]: Same thing here, `#login-container` is commented out so `loginLinkElement` is `null`.

**Review by astrelni — DISMISSED** [review]

File: resume-buddy/src/main/webapp/scripts/script.js (hunk 4 above)
Comment (shreyabarua) [reply resume-buddy/src/main/webapp/scripts/script.js:27]: Yup sorry I moved these elements in the HTML and forgot to delete the commented out version, I'll do that now!

File: resume-buddy/src/main/webapp/styles/style.css [hunk 5]
```diff
@@ -31,3 +31,7 @@ body {
   width: auto;
   text-align: center;
 }
+
+#head {
+    text-align: right;
```
Comment (oydang) [inline resume-buddy/src/main/webapp/styles/style.css]: Nit: indentation should be 2 spaces

**Review by oydang — APPROVED** [review]

**Pull request merged**